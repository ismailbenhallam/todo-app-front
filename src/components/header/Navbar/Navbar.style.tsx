import styled from "styled-components";

export const Nav = styled.nav`
  height: 65px;
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 30px 0 80px;
  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.07);
`;

export const RightContainer = styled.div`
  display: flex;
  justify-content: flex-end;
  align-items: center;
  flex-direction: row;
  width: max-content;
`;
