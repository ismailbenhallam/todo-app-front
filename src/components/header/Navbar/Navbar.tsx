import LanguageChooser from "../LanguageChooser";
import Logo from "../Logo";
import LogoutButton from "../LogoutButton/LogoutButton";
import { Nav, RightContainer } from "./Navbar.style";

export default function Navbar() {
  return (
    <Nav>
      <Logo />
      <RightContainer>
        <LanguageChooser />
        <LogoutButton />
      </RightContainer>
    </Nav>
  );
}
