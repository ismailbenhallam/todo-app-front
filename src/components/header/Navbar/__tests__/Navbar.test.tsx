import renderer from "react-test-renderer";
import Navbar from "../";

test("Navbar should match snapshot", () => {
  const tree = renderer.create(<Navbar />).toJSON();
  expect(tree).toMatchSnapshot();
});
