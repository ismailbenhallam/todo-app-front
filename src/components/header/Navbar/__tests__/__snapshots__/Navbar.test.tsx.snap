// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should match snapshot 1`] = `
<nav
  className="sc-dlfnuX dzXmwp"
>
  <img
    alt="Todo App Logo"
    className="sc-gsTEea HSbWg"
    src="logo.webp"
  />
  <button
    className="sc-bdfBQB jPjxVN"
  >
    DÃ©connexion
  </button>
</nav>
`;
